src/counterfactuals/Counterfactuals.jl

```diff
@@ -2,6 +2,21 @@ using .DataPreprocessing
 using .GenerativeModels
 using .Generators
 using .Models
+using ChainRulesCore
+using Flux
+using MLUtils
+using Plots
+using SliceMap
+using Statistics
+using StatsBase
```

Comment:
you should specify the exact functions that you are importing e.g.
`using MLUtils: transform`

Reply:
Very good point, I'll change that right away.

Reply:
For future reference:

Me and Vincent talked about this and it seems that the preferred use of using is the way I have in my PR currently. using:MLUtils:transform and MLUtils.transform cannot be used at the same time, we have opted to go for MLUtils.transform as it offers better readability.